frontend/src/metabase/components/CollapseSection.jsx

```diff
@@ -0,0 +1,62 @@
+/* eslint "react/prop-types": 2 */
+
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+import Icon from "metabase/components/Icon";
+
+function CollapseSection({
+  children,
+  header,
+  headerClass,
+  bodyClass,
+  initialState = "closed",
+}) {
+  const [isOpen, setIsOpen] = useState(initialState === "open");
```

Comment:
A possible alternative for the name: isExpanded (match with the name, "Collapse"-Section).

Reply:
makes much more sense 👍